pkg/metrics/server.go

```diff
@@ -8,6 +8,7 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
+	ntoconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
```

Comment:
misplaced import

Reply:
you didn't find because there is not, it was a convention enforced in review. You mentioned also a very good way for us to automate this check, we will implement. Thanks!